pkg/pxc/users/users.go

```diff
@@ -8,6 +8,22 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	UserRoot         = "root"
+	UserOperator     = "operator"
+	UserMonitor      = "monitor"
+	UserXtrabackup   = "xtrabackup"
+	UserReplication  = "replication"
+	UserProxyAdmin   = "proxyadmin"
+	UserPMMServer    = "pmmserver"
+	UserPMMServerKey = "pmmserverkey"
+	UserClustercheck = "clustercheck"
```

Comment:
does it make sense to remove the `User` prefix if we going to keep them in `users` package?

Reply:
We think very similarly :) Yes, removed it already.